estate/models/estate_property.py

```diff
@@ -5,13 +5,35 @@ class EstateProperty(models.Model):
     _name = "estate.property"
     _description = "Real state property"
 
-    name = fields.Char(required=True)
+    active = fields.Boolean(default=True)
+    states = fields.Selection(
+        string="Status",
+        selection=[
+            ("new", "New"),
+            ("offer received", "Offer Received"),
+            ("offer accepted", "Offer Accepted"),
+            ("sold", "Sold"),
+            ("canceled", "Canceled"),
+        ],
+        required=True,
+        copy=False,
+        default="new",
+        help="Refers to the status of the current property",
+    )
+    name = fields.Char(required=True, default="My new house")
     description = fields.Text()
+    property_type_id = fields.Many2one(
+        comodel_name="estate.property.type",
+        string="Property type"
+    )
     postcode = fields.Char()
-    date_availability = fields.Datetime()
+    date_availability = fields.Datetime(
+        default=lambda self: fields.Datetime.add(fields.Datetime.today(), months=3),
+        copy=False,
+    )
     expected_price = fields.Float(required=True)
-    selling_price = fields.Float()
-    bedrooms = fields.Integer()
+    selling_price = fields.Float(read_only=True, copy=False)
```

Comment:
```suggestion
    selling_price = fields.Float(readonly=True, copy=False)
```

That was my mistake :sweat_smile: